pkgs/development/python-modules/measurement/default.nix

```
@@ -1,15 +1,27 @@
-{ lib, fetchPypi, buildPythonPackage, pbr, six, sympy }:
+{ lib, fetchFromGitHub, buildPythonPackage, isPy3k
+, sympy, pytest, pytestrunner, sphinx, setuptools_scm }:
 
 buildPythonPackage rec {
   pname = "measurement";
-  version = "2.0.1";
+  version = "3.0.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "36ca385ffdccf140a75a7e1d816a4df97a6dd255f16fd2f53dd7ab43632a8835";
+  # N.B. the pypi src doesn't have the tests!
+  src = fetchFromGitHub {
+    owner = "coddingtonbear";
+    repo = "python-measurement";
+    rev = version;
+    sha256 = "07vs5kfags5v6i4i1ib8vknny5rg9x0nx36n5vklwbhn8nbhyghr";
   };
 
-  propagatedBuildInputs = [ pbr six sympy ];
+  postPatch = ''
+    sed -i 's|use_scm_version=True|version="${version}"|' setup.py
+  '';
+
+  checkInputs = [ pytest pytestrunner ];
+  buildInputs = [ sphinx setuptools_scm ];
```

move to nativeBuildInputs
```suggestion
  buildInputs = [ sphinx ];
```